homeassistant/components/samsungtv/strings.json

```diff
@@ -10,13 +10,12 @@
         }
       },
       "confirm": {
-        "title": "Samsung TV",
-        "description": "Do you want to set up Samsung TV {model}? If you never connected Home Assistant before you should see a popup on your TV asking for authorization. Manual configurations for this TV will be overwritten."
+        "description": "Do you want to set up Samsung TV {model}? If you never connected Home Assistant before you should see a popup on your TV asking for authorization."
       }
     },
     "abort": {
       "already_in_progress": "Samsung TV configuration is already in progress.",
```

Comment:
You could use even more global translations: https://github.com/djpremier/core/blob/a73daa5f20a2c51cbba94223a682e795302929fb/homeassistant/strings.json#L60

Reply:
Welcome @escond! 
I would love if you would finalize it, after all is your work ;-)